manifest.php

```diff
@@ -47,13 +47,13 @@
     'label'       => 'QTI test model',
     'description' => 'TAO QTI test implementation',
     'license'     => 'GPL-2.0',
-    'version'     => '37.3.1',
+    'version'     => '37.3.2',
```

Comment:
Shouldn't it be minor version raised, as dependency raised minor version too ? 
```suggestion
    'version'     => '37.4.0',
```

Reply:
this is a bug fix, in generis we added a feature